spec/controllers/patient_cards_controller_spec.rb

```diff
@@ -1,54 +1,172 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
 
 RSpec.describe PatientCardsController, type: :controller do
+  it 'has a valid factory' do
+    expect(FactoryBot.create(:appointment)).to be_valid
```

Comment:
no point checking if it's valid if you're creating the appointment few lines after it